### PR TITLE
Written configure-porter-in-bgp-mode-en.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @FeynmanZhou @duanjiong @zheng1

### CODE REVIEW

File: doc/configure-porter-in-bgp-mode.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        family:
+          afi: AFI_IP
+          safi: SAFI_UNICAST
+        enabled: true
+      addPaths:
+        config:
+          sendMax: 10
+  nodeSelector:
+    matchLabels:
+      porter.kubesphere.io/rack: master1
```
Comment (duanjiong) [inline doc/configure-porter-in-bgp-mode.md:56]: Use tor or other switch terminology to avoid user misunderstanding

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**